src/models/enums/QuestionType.py

```diff
@@ -2,14 +2,17 @@
 
 
 class QuestionType(Enum):
+    checkbox = auto()
     communicationsOptIn = auto()
     demographic = auto()
     dietaryRestriction = auto()
     email = auto()
     essay = auto()
+    firstName = auto()
     ignore = auto()
+    lastName = auto()
     link = auto()
     resumeLink = auto()
     resumeSharingOptIn = auto()
     shirtSize = auto()
-    checkbox = auto()
+    university = auto()
```

Comment:
remember to update the sample csv and reload, or else the main page will not return any results